src/components/molecules/DataSourceOptions.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { Dispatch, SetStateAction, useMemo } from 'react';
+import { Heading, Pane } from 'evergreen-ui';
+import { DataSourceID, DataSourceVariables } from '../../types/DataSource';
+import MunicipalityEditor from './MunicipalityEditor';
+import { SSBPopulationVariables } from '../../queries/populationInNorway';
+import LatLonInputSet from '../atoms/LatLonInputSet';
+import { MetAPIVariables } from '../../queries/metApi';
+
+type DataSourceOptionsProps = {
+    state: DataSourceVariables;
+    setState: Dispatch<SetStateAction<DataSourceVariables>>;
+
```

Comment:
```suggestion
```

Reply:
It is intentional to separate the two segments (state+setState from other props).